v23.1/create-changefeed.md

```diff
@@ -102,11 +102,11 @@ Example of a Google Cloud Pub/Sub sink URI:
 
 The following are example file URLs for each of the cloud storage schemes:
 
-Location     | Example                                                                          
+Location     | Example
 -------------+----------------------------------------------------------------------------------
-Amazon S3    | `'s3://{BUCKET NAME}/{PATH}?AWS_ACCESS_KEY_ID={KEY ID}&AWS_SECRET_ACCESS_KEY={SECRET ACCESS KEY}'`  
+Amazon S3    | `'s3://{BUCKET NAME}/{PATH}?AWS_ACCESS_KEY_ID={KEY ID}&AWS_SECRET_ACCESS_KEY={SECRET ACCESS KEY}'`
 Azure Blob Storage | `'azure://{CONTAINER NAME}/{PATH}?AZURE_ACCOUNT_NAME={ACCOUNT NAME}&AZURE_ACCOUNT_KEY={URL-ENCODED KEY}'`
-Google Cloud | `'gs://{BUCKET NAME}/{PATH}?AUTH=specified&CREDENTIALS={ENCODED KEY'`                         
+Google Cloud | `'gs://{BUCKET NAME}/{PATH}?AUTH=specified&CREDENTIALS={ENCODED KEY'`
 HTTP         | `'http://localhost:8080/{PATH}'`
```

Comment:
Not for this PR, but should we recommend https? Is it supported?

Reply:
`https` is not, but we do have `webhook-https` and 23.1 brought about performance improvements for the webhook sink. Right now `http` is mostly used because the webhook was not as performant. As you say, I think we can look at updating this now to recommend webhook over `http`.